src/makernote_int.cpp

```diff
@@ -483,7 +483,7 @@ namespace Exiv2 {
         // Todo: This removes any gap between the header and
         // makernote IFD. The gap should be copied too.
         TiffHeader th(byteOrder);
-        DataBuf buf = th.write();
+        const DataBuf& buf = th.write();
```

Comment:
Same question here.  The signature of write is:

```
        virtual DataBuf write() const;
```

Why to add the `const & ` ??

Reply:
Same as above. I'll revert.